frontend/src/components/Items.vue

```diff
@@ -127,13 +142,43 @@ export default {
       },
       immediate: true,
     },
+    items: {
+      handler: function () {
+        if (
+          this.items.some((item) => item.hasOwnProperty("activeState")) &&
+          this.mode === 'single'
+        ) {
+          let activeItem = null;
+          for (let i = 0; i < this.items.length; i++) {
+            const item = this.items[i];
+            if (item.activeState) {
+              activeItem = i;
+              break;
+            }
+          }
+          this.expandedPanels = activeItem;
+        }
+      },
+      deep: true,
+      immediate: true,
+    },
+  },
+  updated() {
+    if (
+      this.items.some((item) => item.hasOwnProperty("activeState")) &&
```

Comment:
what is the condition `item.hasOwnProperty("activeState")` guarding against?
It seems a little strange to only "update" the activeState property if such a property
already exists. 

I assume this `items` are populated by contributors to the framework?
If so would they have to remember to initialize `activeState` to something (true/false)
for this logic  to work successfully?